zerver/models.py

```diff
@@ -289,6 +295,13 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
     # at `foo.example.com`.
     string_id = models.CharField(max_length=MAX_REALM_SUBDOMAIN_LENGTH, unique=True)
 
+    # uuid and a secret for the sake of per-realm authentication with the push notification
+    # bouncer.
+    uuid = models.UUIDField(default=uuid4, unique=True)
+    uuid_owner_secret = models.CharField(
+        max_length=41, default=generate_realm_uuid_owner_secret, unique=True
```

Comment:
Do we want to just use `TextField` for these? I seem to recall there not being a lot of benefit to fixed-width `CharField` in modern postgres.

Reply:
Hmm, no strong feelings here. We seem to be kind of randomly using `TextField` vs `CharField`, though our "api_key", "token" etc. kind of columns are `CharField`, so I went with that for consistency.

Reply:
I suppose `TextField` at least lets us not have to think about what the length of the secret becomes based on the `zuliprealm_` or whatever prefix. Pushed with this change, and also dropped the `unique=True` on the secret.

Reply:
Yeah, I think the pattern is the newer ones are `TextField` but we haven't bothered changing the older ones, since `user_profile.api_key` is likely to be replaced with creating a separate `ApiKey` table.